Packs/Perch/Integrations/Perch/Perch.py

```diff
@@ -433,8 +441,15 @@ def create_indicator_command():
 def fetch_alerts(last_run, headers):
     last_fetch = last_run.get('time')
     url = '/alerts'
-    res = http_request('GET', url, headers=headers)
-    items = res.get('results')
+    statuses_to_fetch = demisto.params().get('soc_status', [])
+    if statuses_to_fetch:
+        items = []
+        for status in statuses_to_fetch:
```

Comment:
is there no way to fetch several statuses with a single call?

Also if there is a get-alerts style command we should make sure it supports status filtering as well.

Reply:
No, the API gets the parameter only as a string.

There is a command *perch-search-alerts* and it's supports the soc_status filtering.